src/pages/Dashboard.tsx

```diff
@@ -85,33 +84,41 @@ export default function Dashboard() {
             setPath([{ name: 'Dashboard', path: '/dashboard' }]);
         }
     }, []);
-
-    const categoryBlockStyle = {
-        margin: '10px',
-        padding: '10px',
-    };
-
+    
     return (
         <div>
-            <Typography variant='h1'>Institutions</Typography>
-            <div style={categoryBlockStyle}>
-                <Typography variant='h3'>My Organizations</Typography>
-                <div style={{ display: 'flex' }}>
-                    {renderOrganizations(adminOrganizations, true)}
-                </div>
-            </div>
-            <div style={categoryBlockStyle}>
-                <Typography variant='h3'>Organizations with Access</Typography>
-                <div style={{ display: 'flex' }}>
-                    {renderOrganizations(accessOrganizations, true)}
-                </div>
-            </div>
-            <div style={categoryBlockStyle}>
-                <Typography variant='h3'>All Organizations</Typography>
-                <div style={{ display: 'flex' }}>
-                    {renderOrganizations(noAccessOrganizations, false)}
-                </div>
-            </div>
+            <Typography variant='h1' style={{marginBottom: 20}}>Institutions</Typography>
+            {
+                adminOrganizations.length > 0 ?
+                <>
+                    <Typography variant='h3'>My Organizations</Typography>
+                    <div style={{ display: 'flex' }}>
+                        {renderOrganizations(adminOrganizations, true, 'Admin')}
+                    </div>
+                </>
+                : null
+            }
```

Comment:
nit: could be refactor to this
```suggestion
            { adminOrganizations.length && 
               <>
                    <Typography variant='h3'>My Organizations</Typography>
                    <div style={{ display: 'flex' }}>
                        {renderOrganizations(adminOrganizations, true, 'Admin')}
                    </div>
                </>
            }
```